apidoc/Annotation.yml

```diff
@@ -325,3 +325,21 @@ properties:
     since: "6.2.0"
     type: Titanium.UI.iOS.PreviewContext
     osver: {ios: {min: "9.0"}}
+
+methods:
+
+  - name: animate
+    summary: Animate annotation to new location.
+    parameters:
+        - name: newLocation
+          summary: latitude and longitude where annotation will animate.
+          type: Array<Number>
+    platforms: [iphone, ipad]
```

Comment:
A `since` property needs to be set on new APIs. This should be set to the SDK version we're expecting to ship the changes in.